.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: generic
+
+services:
+  - docker
+
+before_install:
+  - docker-compose up -d
+  - docker-compose ps
+  - docker ps -a
+
+script:
+  - docker-compose exec truffle yarn run test:docker
+
+notifications:
+  slack: hive-ida:MXjYRdIBifAJ85SMx5uGWrMu
```

Comment:
Is this meant to be public? Or needs to use travis encryption!?

Reply:
Oops. Regenerated and encrypted key.